src/InstrumentationEngine.Attach/Program.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Json;
+using Microsoft.Diagnostics.NETCore.Client;
+using static System.FormattableString;
+
+namespace Microsoft.InstrumentationEngine
+{
+    internal sealed class Program
+    {
+        private static readonly Guid InstrumentationEngineClsid = new Guid("324F817A-7420-4E6D-B3C1-143FBED6D855");
+
+        internal static void Main(string[] args)
+        {
+            // TODO: Parse command line for additional arguments such as configuration files
+            if (args.Length != 1)
```

Comment:
>!= [](start = 28, length = 2)

< 1 ?

Reply:
It's also an error if there is more than one argument as it is currently parsed.

---
In reply to: [384688540](https://github.com/microsoft/CLRInstrumentationEngine/pull/212#discussion_r384688540) [](ancestors = 384688540)